bbc/simorgh
# WSTEAM1-725 CPS SPIKE:Live page archive support

Resolves JIRA [WSTEAM1-725](https://jira.dev.bbc.co.uk/browse/WSTEAM1-725)

Overall changes
======
Routes CPS live pages to be rendered by Simorgh using a query param override i.e the `renderer_env`.

Code changes
======
- creates `arabic` live page data fixture
- Seperates logic for `LIVE` and `TOPIC` pages inside of `constructPageFetchUrl` function (previously they were grouped together), and refactors existing logic for when the environment is local.
- Updates existing test cases.
- Adds integration tests.
- Updates existing snapshots.

Testing
======
1. Pull down PR
2. go to http://localhost:7081/arabic/live/67574192?renderer_env=live and see it is [this cps live page ](https://www.test.bbc.com/arabic/live/67574192) being rendered by simorgh
3. Similarly go to http://localhost:7081/serbian/lat/live/media-23179005 which has been added as a local fixture using https://www.test.bbc.com/serbian/lat/live/media-23179005?morph_env=test

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Gap analysis between Simorgh and CPS render](https://paper.dropbox.com/doc/Spike-Live-Page-gap-analysis-Simorgh-vs-CPS-Live--CGCg5Marbmh0IYCImBA3JHJnAg-e162RpJOpwUnNJF55qzws)

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
